nuitka/tree/Operations.py

```diff
@@ -73,18 +74,13 @@ def onEnterNode(self, node):
     def _onEnterNode(self, node):
         if node.isExpressionBuiltinImport():
 
-            for (
-                used_module_name,
-                used_module_filename,
-                finding,
-                level,
-            ) in node.getUsedModules():
+            for module in node.getUsedModules():
                 self.used_modules.add(
```

Comment:
Same here, you are getting attempt objects to create new ones, which should be unneccessary.